recipe/meta.yaml

```diff
@@ -37,6 +37,77 @@ requirements:
     - typing_extensions >=4.6 # [py<312]
 
 test:
+  imports:
+    - IPython
```

Comment:
Are these `test.imports` meant to replace `run_test.py`?  Because it seems to still be running `run_test.py`.

Reply:
Well, I attempted to replace `run_test.py` because it seems nothing is being tested for `win` or `linux` but I wasn't sure if it was worth the effort. I left the imports just in case someone else decided to pick up where I left off.

Reply:
As we have `run_test.py` I would move all these imports there

Reply:
@Jrice1317 if `run_test.py` is being successfully run on all the systems then I would remove these `test.imports` as it gives the impression they should be being run when they aren't.  And simple folk like myself then get confused!